irohad/model/converters/impl/pb_query_response_factory.cpp

```diff
@@ -317,6 +351,43 @@ namespace iroha {
         }
         return pb_response;
       }
+
+      model::ErrorResponse PbQueryResponseFactory::deserializeErrorResponse(
+          const protocol::ErrorResponse &response) const {
+        model::ErrorResponse result{};
+        switch (response.reason()) {
+          case protocol::ErrorResponse_Reason_STATELESS_INVALID:
+            result.reason = ErrorResponse::STATELESS_INVALID;
+            break;
+          case protocol::ErrorResponse_Reason_STATEFUL_INVALID:
+            result.reason = ErrorResponse::STATEFUL_INVALID;
+            break;
+          case protocol::ErrorResponse_Reason_NO_ACCOUNT:
+            result.reason = ErrorResponse::NO_ACCOUNT;
+            break;
+          case protocol::ErrorResponse_Reason_NO_ACCOUNT_ASSETS:
+            result.reason = ErrorResponse::NO_ACCOUNT_ASSETS;
+            break;
+          case protocol::ErrorResponse_Reason_NO_ACCOUNT_DETAIL:
+            result.reason = ErrorResponse::NO_ACCOUNT_DETAIL;
+            break;
+          case protocol::ErrorResponse_Reason_NO_SIGNATORIES:
+            result.reason = ErrorResponse::NO_SIGNATORIES;
+            break;
+          case protocol::ErrorResponse_Reason_NOT_SUPPORTED:
+            result.reason = ErrorResponse::NOT_SUPPORTED;
+            break;
+          case protocol::ErrorResponse_Reason_NO_ASSET:
+            result.reason = ErrorResponse::NO_ASSET;
+            break;
+          case protocol::ErrorResponse_Reason_NO_ROLES:
+            result.reason = ErrorResponse::NO_ROLES;
+            break;
+          default:
+            break;
```

Comment:
why empty default?

https://github.com/hyperledger/iroha/pull/869/files#diff-54506143c6fcf71e8b5954f1e82bf10fR118
Here it is not empty

Reply:
This legacy code was extended only for test purposes. This will be replaced with shared model soon.